### PR TITLE
fix(Accordion): L3-2559 Build Accordion component QA test fix

### PR DESCRIPTION
**Pull request by zhenpanda**
**Jira ticket**

(https://phillipsauctions.atlassian.net/browse/L3-2559)

**Summary**

Fixes for the accordion component:
1. All items are open-able independently by default.
2. Plus/Minus Icon display fixed.
3. Updated lock icon to the correct Figma icon seem here:  (https://www.figma.com/design/hMu9IWH5N3KamJy8tLFdyV/EASEL-Compendium%3A-Tokens%2C-Components-%26-Patterns?node-id=4433-163013&t=UXNWH94WW2SilZG5-0)

**Change List (describe the changes made to the files)**

- Additions, Changes, and Removals

**Acceptance Test (how to verify the PR)**

- Add step by step instructions on how to verify the change

**Regression Test**

- (Optional) Add verification steps to make sure this PR doesn't break old functionality

**Evidence of testing**

https://github.com/user-attachments/assets/7bfed068-d18b-44aa-8882-b030bb83683e

**Things to look for during review**

- [ ] PR title should correctly describe the most significant type of commit. I.e. `feat(scope): ...` if a `minor` release should be triggered.
- [ ] All commit messages follow convention and are appropriate for the changes
- [ ] All references to `phillips` class prefix are using the prefix variable
- [ ] All major areas have a `data-testid` attribute.
- [ ] Document all props with jsdoc comments
- [ ] All strings should be translatable.
- [ ] Unit tests should be written and should have a coverage of 90% or higher in all areas.

New Components

- [ ] Are there any [accessibility considerations](https://www.w3.org/WAI/ARIA/apg/patterns/) that need to be taken into account and tested?
- [ ] Default story called "Playground" should be created for all new components
- [ ] Create a jsdoc comment that has an Overview section and a link to the Figma design for the component
- [ ] Export the component and its typescript type from the `index.ts` file
- [ ] Import the component scss file into the `componentStyles.scss` file.

### CODE REVIEW

**Review by Zach-OfAllTrades — APPROVED** [review]
Good stuff!

File: src/components/Accordion/Accordion.tsx [hunk 1]
```diff
@@ -27,15 +23,15 @@ export interface AccordionProps extends React.HTMLAttributes<HTMLDivElement> {
  *
  * [Storybook Link](Point back to yourself here)
  */
-const AccordionComponent = ({ className, type = 'single', children, ...props }: AccordionProps) => {
+const accrdionType = 'multiple';
```
Comment (aahill50) [inline src/components/Accordion/Accordion.tsx:26]: you have a typo in the const name:
```suggestion
const accordionType = 'multiple';
```

**Review by aahill50 — APPROVED** [review]
Looks good, just one small correction to make

**Pull request merged**